dracut/30ignition/module-setup.sh

```diff
@@ -59,6 +59,7 @@ install() {
             "$systemdsystemunitdir/ignition-$x.target"
     done
 
+    install_ignition_unit ignition-setup.service ignition-diskful.target
```

Comment:
As currently structured, we're diverging from FCOS by not supporting:

- Per-platform Ignition base configs in live boots (currently unused functionality)
- Supplying an Ignition config via an additional initramfs archive